src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/telemetry/telemetry.component.html

```diff
@@ -0,0 +1,300 @@
+<cd-loading-panel *ngIf="loading && !error"
+                  i18n>Loading configuration...</cd-loading-panel>
+<cd-alert-panel type="error"
+                *ngIf="loading && error"
+                i18n>The configuration could not be loaded.</cd-alert-panel>
+
+<div class="cd-col-form"
+     *ngIf="!loading && !error">
+  <ng-container [ngSwitch]="step">
+    <!-- Configuration step -->
+    <div *ngSwitchCase="1">
+      <form name="form"
+            #formDir="ngForm"
+            [formGroup]="configForm"
+            novalidate>
+        <div class="card">
+          <div class="card-header"
+               i18n>Step {{ step }} of 2: Telemetry report configuration</div>
+          <div class="card-body">
+            <p i18n>The telemetry module sends anonymous data about the cluster back to the Ceph developers
+              to help understand how Ceph is used and what problems users may be experiencing.<br/><br/>
+              The data being reported does <b>not</b> contain any sensitive data like pool names, object names, object contents,
+              hostnames, or device serial numbers. It contains counters and statistics on how the cluster has been
+              deployed, the version of Ceph, the distribution of the hosts and other parameters which help the project
+              to gain a better understanding of the  way Ceph is used. The data is sent secured to https://telemetry.ceph.com.</p>
```

Comment:
How about adding a note that the user can preview the data that is submitted on the next page prior to enabling this functionality?